### PR TITLE
Atlantic 2024 CFP dates and text 

### PR DESCRIPTION
**Pull request by plaindocs**
- [x] update CFP text
- [ ] write blog 

----
📚 Documentation preview 📚: https://writethedocs-www--2096.org.readthedocs.build/

### CODE REVIEW

**Comment by hzoppetti** [comment]
One of the things I liked (and don't mind stealing) when looking around at other CFPs was the wording about benefits to the industry. For example, I think we could add verbiage in the abstract section about making sure to add how your talk will benefit your intended audience. 

Also, I still think the proposal could use a fourth section for synopsis. This would also require a change to the pretalx form.
However, this section would force the submitter to think through their talk a little bit more and give a brief outline stating the flow of their talk and would give reviewers a better idea of the topics being covered.

**Comment by jputrino** [comment]
I’d like to see us go farther with the example abstracts. We could link to examples from past conferences, or put together a demo example — something people can copy and paste and then adjust for their own talk. 
If we want to go with the latter, I can help put it together.

**Comment by plaindocs** [comment]
Hey folks, thanks

@hzoppetti would you like to have a bash at drafting those? 

@jputrino what would you see that isn't in https://www.writethedocs.org/conf/atlantic/2024/example-proposal/ (though now I look at it the formatting is a little off there). I think this is better than giving specific examples from past years. Edits super welcome.

@janine-c a checkbox saying "I've read this entire doc" right? :japanese_ogre: 

My list from slack

- [ ] more spoilers
- [ ] show you're aware of what we've already had on stage
- [ ] dial down tooling

**Comment by hzoppetti** [comment]
Hmm - I tried to push a commit to this PR, but I got a 403. I could fork. Do you do use a forking git workflow? Or do you want me to paste some comments below?

**Comment by plaindocs** [comment]
Hey @hzoppetti sorry about that. Yeah, either a fork, or comments below as you prefer. I'll go poking and add you to the org.

**Comment by hzoppetti** [comment]
I feel some of the file could benefit from some clean up. For example, the last line below should be added to the "Other Information" section as well. And stylistically, I would choose to remove many uses of the word "that".  

For my "commit", I added the following to example-proposal.rst
In the bulleted list under abstract:

* The benefits your talk might bring to the documentation industry

and a new section at the bottom:

Synopsis
--------

Provide a brief outline of your talk. This can be as simple as a bulleted list.

The reviewers want to see you have thought carefully about your talk, the ideas you will cover, and you have enough (but not too much) information to fill the time.

For example:

* Introduction
* Problems we faced: Problem statement
* Approaches we proposed and how we proposed them
* Solution we tried 1
* Metrics for solution 1
* Lessons we learned from solution 1
* Solution we tried 2
* Metrics for solution 2
* Lessons we learned from solution 2
* Ultimate outcome
* Celebration
* Where to find me

Only the proposal committee sees the information you include here.

**Comment by plaindocs** [comment]
Hey folks, I've tweaked this and am fairly happy with the improvements. Any final thoughts? 
I didn't ask for the full outline, as I think most folks won't have it ready when they are submitting.

**Comment by plaindocs** [comment]
@mxsasha anything you want to add to the blog post?

File: docs/conf/atlantic/2024/news/announcing-cfp.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Sign up to the `Atlantic conference mailing list <https://www.writethedocs.org/newsletter/>`_ to be the first to hear details as we announce them.
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+**The conference is virtual and all talks will be pre-recorded, allowing you to present and attend from anywhere!**
+Our video producer will be available to help with recording if you want it.
```
Comment (mxsasha) [inline docs/conf/atlantic/2024/news/announcing-cfp.rst]:
```suggestion
Our video producer will be available to help with recording, at your request.
```

**Review by mxsasha — APPROVED** [review]
One minor nit. These changes to the CFP page are 💯

File: docs/conf/atlantic/2024/news/announcing-cfp.rst (hunk 1 above)
Comment (mxsasha) [reply docs/conf/atlantic/2024/news/announcing-cfp.rst]: Or something else. "if you want it" just felt out of tone with the rest of it

**Pull request merged**